### PR TITLE
[#205] Add all existing run parameters to README

### PR DESCRIPTION
**Pull request by pnatashap**
Run parameters are fixes
All parameters from the code are described

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,35 +69,59 @@ Run it locally and read its output:
...
+
+Full command format is (all parameters are optional):
+```bash
+$ pdd [--verbose] [--quiet] [--remove] [--skip-gitignore] [--skip-errors] \
+      [--source <project_dir_path>] [--file puzzles_file.xml] [--include src/**/*.py] \
+      [--format xml|html] [--rule min-words:5] [--exclude src/**/*.java]
 ```
+| Parameter               | Description                                                                           |
+|-------------------------|---------------------------------------------------------------------------------------|
+| --verbose               | Enable verbose (debug) mode. --file must be used in case of using                     |
```
Comment (php-coder) [inline README.md:111]: `in case of using` .... ?

File: README.md [hunk 2]
```diff
@@ -38,35 +69,59 @@ Run it locally and read its output:
...
+| --quiet                 | Disable logs                                                                          |
+| --remove                | Remove all found puzzles from the source code                                         |
+| --skip-gitignore        | Don't look into .gitignore for excludes                                               |
+| --skip-errors           | Suppress error as warning and skip badly formatted puzzles (do not skip broken rules) |
+| --source <project-path> | Source directory to parse ("." by default)                                            |
+| --file puzzles.xml      | File to save report into into (xml of html) (displayed in console by default)         |
```
Comment (php-coder) [inline README.md:117]:
```suggestion
| --file puzzles.xml      | File to save report into (xml of html) (displayed in console by default)         |
```

File: README.md [hunk 3]
```diff
@@ -38,35 +69,59 @@ Run it locally and read its output:
...
+| --include *.py          | Glob pattern to exclude (can be used several times)                                   |
```
Comment (php-coder) [inline README.md:118]:
```suggestion
| --include *.py          | Glob pattern to include (can be used several times)                                   |
```

File: README.md [hunk 4]
```diff
@@ -38,35 +69,59 @@ Run it locally and read its output:
...
+| --exclude *.java        | Glob pattern to include (can be used several times)                                   |
```
Comment (php-coder) [inline README.md:119]:
```suggestion
| --exclude *.java        | Glob pattern to exclude (can be used several times)                                   |
```

File: README.md [hunk 5]
```diff
@@ -167,7 +222,7 @@ Here is a list of rules available now:
   This rule is used by default and you can't configure it at the moment,
   it must always be set to `1`.
 
-You can put all command line options into `.pdd` file. The options from the
+:bulb: You can put all command line options into `.pdd` file. The options from the
```
Comment (php-coder) [inline README.md:225]: Nice! I didn't know about that emoji..

File: README.md (hunk 1 above)
Comment (pnatashap) [reply README.md:111]: this option... will add

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**